tests/test_pytest_plugin.py

```diff
@@ -89,36 +89,36 @@ async def test_async_session_fixture_dont_patch_engine_writes_in_db(
 
 @mark.sqlalchemy("1.4")
 def test_all_opened_sessions_are_within_the_same_transaction(
-    sqla_connection, session, session_factory, singer_cls
+    session, session_factory, singer_cls
 ):
     session.add(singer_cls(id=1, name="Bob Marley", country="Jamaica"))
     session.commit()
 
-    other_session = session_factory(bind=sqla_connection)
```

Comment:
The session factory now automatically binds the sqla_connection, so this is no longer necesary